src/context/Web3Context.tsx

```diff
@@ -17,11 +17,14 @@ export type OnboardConfig = Partial<Omit<Initialization, 'networkId'>>;
 type EthGasStationSettings = 'fast' | 'fastest' | 'safeLow' | 'average';
 type EtherchainGasSettings = 'safeLow' | 'standard' | 'fast' | 'fastest';
 
+
+
 type TokenConfig = {
   address: string;
   name?: string;
   symbol?: string;
   imageUri?: string;
+  attachFunctions?: boolean
```

Comment:
what is the reasoning behind this parameter being passed in?

Reply:
In some instances there might not be either a need for passing functions through or that the context could be used without there being a User controlled wallet provider fetching the token data

Reply:
unnecessary overcomplication... if the erc address is being passed in the functions should always be available